docs/applications/voip/setup-a-mumble-server-on-debian.md

```diff
@@ -0,0 +1,154 @@
+---
+author:
+    name: Linode Community
+    email: docs@linode.com
+description: 'Setting up Mumble on Debian'
+keywords: 'mumble, debian, murmur, open source, gaming, VOIP, voice chat'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 'Thursday, September 24th, 2015'
+modified: Thursday, September 24th, 2015
+modified_by:
+    name: Linode
+title: 'Mumble on Debian'
+contributor:
+    name: Your Name
+    link: Github/Twitter Link
+---
+
+Mumble is an open-source VoIP client, designed for gamers, that requires a server for all clients to connect to. This guide instructs you on how to install and configure the Mumble server (also called Murmur) on Debian.
+
+No changes need to be made to the system beforehand if you follow this guide. 
+
+Througout this guide, all commands are run as root.  If you are not root user, you can prepend each command with "sudo." 
+
+## Server Side
+
+### Install and Simple Setup
+
+1. Since Murmur is in the official Debian repositories, you can just use apt-get to install it.  Be careful though, the package is `mumble-server` and not `murmur`. Enter the following command (remember to prepend with sudo if needed):
+
+	apt-get install mumble-server
+
+2. After installation, you can use `dpkg-reconfigure` to configure the initial setup.
+
+The first question asked during system config will be whether or not you want the server to run at boot.  This is identical to using the command `systemctl enable mumble-server`.
```

Comment:
This is true for Debian 8 but not Debian <8. The other commands below are for both versions, so I assume you want this one to be correct for older versions as well.